### DISCUSSION TITLE
Need to show timeline like in Devtool waterfall

### DISCUSSION THREAD

**Discussion by AlinaKIRIC** [GoogleChrome/lighthouse#12085] [Q&A]
Hi! I am using lighthouse cli 7.0.1. and mostly I checked metrics manually and with report viewer.
But now, to analyze deeper I need to see waterfall(like in Devtool). I tried to import Json file in Devtool performance tab but it doesn't work.

Probably somebody knows where I can see timeline from json result? I tried to add --save-assets but and check in lighthouse report viewer, but couldn't  find there something that can show me waterfall. 
Maybe for such cases  these is some online site  that I don't know yet. 

I will be very appreciate for help

**Comment by adamraine** [comment]
Running lighthouse with the flags `-GA` will save the trace in `./latest-run/defaultPass.trace.json`. You can import this trace into the DevTools performance panel to view the network graph.

Keep in mind that the lighthouse metrics will not match the trace by default because Lighthouse uses simulated throttling. If you need the metrics to match the results in the trace, run Lighthouse with `--throttling-method=devtools`. Check out our docks for more on this https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#network-throttling.

**Reply by AlinaKIRIC** [reply]
Yes, got it, thank you! 
Also,one small question, with --throttling-method "provided" do I get same results as with "devtools"?

**Reply by adamraine** [reply]
`--throttling-method=provided` will use the network speed and CPU power provided by the environment. `--throttling-method=devtools` will use devtools to actively throttle the page. Both should give you a trace that matches LH metrics.

**Comment by AlexanderChapurin** [comment]
Hello! I am also keen on having a waterfall chart in my LH reports. I am running puppeteer + Lighthouse ,and wonder if there is an option to use the mentioned above flag in my  case?

**Reply by adamraine** [reply]
`-G` and `-A` have equivalent settings that can be defined on the conifg:
https://github.com/GoogleChrome/lighthouse/blob/6e9467cd1ef0193582198cf44786a259a65f85e3/types/lhr/settings.d.ts#L66-L69

**Reply by AlexanderChapurin** [reply]
Many thanks, Adam! Appreciate your help!
